doc/sphinx/source/esmvaldiag/observations.inc

```diff
@@ -1,7 +1,269 @@
-.. _observations:
-
 ************************************************************
 Contributing a CMORizing script for an observational dataset
 ************************************************************
 
+ESMValTool is designed to work with CF compliant data and 
+follows the CMOR tables from the CMIP data request, therefore 
+the observational datasets need to be CMORized for usage in ESMValTool.
+The following steps are necessary to prepare an observational
+data set for the use in ESMValTool.
+
+| `1. Check if your variable is CMOR standard`_
+| `2. Edit your configuration file`_
+| `3. Store your dataset in the right place`_
+| `4. Create a cmorizer for the dataset`_
+| `4.1 Cmorizer script written in python`_
+| `4.2 Cmorizer script written in NCL`_
+| `5. Run the cmorizing script`_
+| `6. Naming convention of the observational data files`_
+| `7. Test the cmorized dataset`_
+
+
+1. Check if your variable is CMOR standard
+==========================================
+
+Most variables are defined in the CMIP data request and can be found in the
+CMOR tables in the folder `/esmvalcore/cmor/tables/cmip5/Tables/
+<https://github.com/ESMValGroup/ESMValCore/tree/development/esmvalcore/cmor/tables/cmip5/Tables>`_,
+differentiated according to the ``MIP`` they belong to. The tables essentially
```

Comment:
Instead of "The tables essentially follow", I think it would be more clear to write "The tables are a copy of "
```